CONTRIBUTING.md

```diff
@@ -59,43 +62,44 @@ Some examples:
 - `mysql {{database_name}}`
 - `unrar x {{compressed.rar}}`
 
-In short, make it as intuitive as possible for the user to figure out
-how to use the command and fill it in with values.
+In short, make it as intuitive as possible
```

Comment:
My interpretation of semantic linebreaks is that lines should be broken in semantically meaningful blocks -- ideally full clauses, but not necessarily.

In this case, the full clause would be "make it as intuitive as possible for the user to figure out how to use the command and fill it in with values", but since that's still too long, I've broken down the sentence to the basic form "make it as intuitive as possible / for the user to / X", and split it in the first slash.

The second most sensible alternative, IMO, would be in the second slash, rather than in the middle of X as it was previously.

Using the comma doesn't help much since we still need to wrap the line, and the clause before the comma is too short for it to be useful to set it apart.

In any case, as @ostera says, this is ultimately not very problematic as the rendered version looks the same regardless of where we break the sentence.